packages/amplify-graphql-transformer-interfaces/src/graphql-api-provider.ts

```diff
@@ -113,13 +114,13 @@ export interface InlineMappingTemplateProvider {
 
```

Comment:
Why doesn't `InlineMappingTemplateProvider` above also need an AssetManagerProvider? Do we not write it as an asset anywhere during processing?

Reply:
The `InlineTemplate` does not store any asset.

https://github.com/aws-amplify/amplify-category-api/blob/a5a233cfc1aa0859a9adf74f319b53c9479b38d4/packages/amplify-graphql-transformer-core/src/cdk-compat/template-asset.ts#L86-L103